2.5/README.md

```diff
@@ -1,9 +1,8 @@
 Ruby 2.5 container image
 ========================
 This container image includes Ruby 2.5 as a [S2I](https://github.com/openshift/source-to-image) base image for your Ruby 2.5 applications.
-Users can choose between RHEL, CentOS and Fedora based builder images.
```

Comment:
Why this change?

It's no longer available in Fedora either (Fedora has Ruby 2.7+), and it is not EOL'd in RHEL.

IOW none of those are available or shouldn't be used for production.

Reply:
Thanks for the review. You are right, there should be only RHEL8.